scripts/autoexec.lua

```diff
@@ -50,7 +52,7 @@ function enableModelAnimReplaceFlags(m_id, copy_meshes_back)
     end;
 end;
 
-if (getLevelVersion() < TR_II) then
+if (ver == TR_I) then
```

Comment:
ver < TR_II should be to cover all TR_I games (TR_I, TR_I_UB, TR_I_DEMO)

Reply:
i will fix it.